src/Phinx/Db/Adapter/PdoAdapter.php

```diff
@@ -393,6 +393,36 @@ public function insert(Table $table, $row)
     /**
      * {@inheritdoc}
      */
+    public function update(Table $table, $row, $primaryKeyName = 'id')
+    {
+        $this->startCommandTimer();
+        $this->writeCommand('update', array($table->getName()));
+
+        // Store primary key for where clause
+        $pKey = $row[$primaryKeyName];
+        // Remove primary key as it doesn't get 'updated'
+        unset($row[$primaryKeyName]);
+
+        $columns = array_keys($row);
+        $sql = sprintf(
+            "UPDATE %s SET ",
+            $this->quoteTableName($table->getName())
+        );
+        $setColumns = [];
+        foreach ($columns as $col) {
+            $setColumns[] = $this->quoteColumnName($col) . '=?';
+        }
+        $sql .= implode(', ', $setColumns);
+        $sql .= " WHERE " . $primaryKeyName . "='" . $pKey . "'";
```

Comment:
1. Does $primaryKeyName needs to be escaped appropriately?
2. Should $pKey be re-added to the end of $row once the update part of the query has been generated? No need to quote a string and if the string contains a `'` ... 💣